common/service/dynamicconfig/constants.go

```diff
@@ -68,7 +68,7 @@ var keys = map[Key]string{
 	EnableNamespaceNotActiveAutoForwarding: "system.enableNamespaceNotActiveAutoForwarding",
 	TransactionSizeLimit:                   "system.transactionSizeLimit",
 	MinRetentionDays:                       "system.minRetentionDays",
-	MaxDecisionStartToCloseSeconds:         "system.maxDecisionStartToCloseSeconds",
+	MaxDecisionTaskStartToCloseTimeout:     "system.maxDecisionTaskStartToCloseTimeout",
```

Comment:
Timeout is just timeout: it should not have `Max` prefix.

Reply:
This is not a timeout. This is the maximum value allowed. If user specifies a higher one it is going to be lowered to the configured maximum.

Reply:
Sounds little weird but makes sense now.

Reply:
Usually setting this timeout to a high value is a mistake. That's why we limit it.